clap_man/src/render.rs

```diff
@@ -0,0 +1,195 @@
+use std::io::Write;
+
+use crate::man::{bold, document_title, escape, indent, italic, list, section_title};
```

Comment:
nit: Personally, I'd prefer just `use crate::man;`.  I know all of clap pretty much imports all but I want to move away from that because
- It makes it easy to have merge conflicts
- It increases likelihood of name collisions
- It can obscure intent (I didn't assumed `document_title` was in `render.rs` rather than `man.rs` when reading the code, meaning I didn't take it to be part of the man format).

The latter I think is especially pertinent here which is why I'm calling this out compared to anything else.  Seeing `man::escape` instead of `escape` adds that little bit of extra context I think makes things more readable.

Reply:
With the latest refactor this now is used via `roff`, though I didn't import them like you mentioned here, not sure what you think about it now that they live in a third-party crate.